index.js

```diff
@@ -0,0 +1,141 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2018,
+    ecmaFeatures: {
+      jsx: true,
+    },
+    sourceType: 'module',
+    project: './tsconfig.json',
+  },
+  extends: ['plugin:react/recommended', 'plugin:prettier/recommended'],
+  plugins: ['import', 'unicorn', 'prettier'],
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  env: {
+    browser: true,
+    es6: true,
+  },
+  globals: {
+    require: true,
+    process: true,
+  },
+  rules: {
+    'default-case': 'error',
+    'import/extensions': [
+      'error',
+      'always',
+      { js: 'never', jsx: 'never', ts: 'never', tsx: 'never', hbs: 'never', vue: 'never' },
+    ],
+    'import/no-unresolved': 'error',
+    'max-lines': 'error',
```

Comment:
Often you want a util file with only named exports, and having just 1 in there at the starts causes you to always disable this rule for that file.

```suggestion
    'import/prefer-default-export': 'off',
    'max-lines': 'error',
```

Preferring named exports over default exports is a whole discussion on its own (certainly has advantages).

Reply:
I also prefer named exports over default exports, so I agree

Reply:
Just a note, I'm currently using the plugin within a project, and works really nice: https://www.npmjs.com/package/eslint-plugin-simple-import-sort which comes like prettier with a predefined set.

Reply:
If you look 1 (outdated) comment up, I added https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/order.md

Is there something that the `simple-import-sort` does better? :)

Reply:
I think it adds grouping, and sorting alphabetically in import `{a, b, c}`

Reply:
@ThaNarie I think your config does some ordering as well, I think `simple-import-sort` just has lot more opinionated settings for ordering imports.

Reply:
I don't mind changing it if we like those results better :)